.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,32 @@
+name: Node CI
+on: [push]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        browser: [ChromeHeadless, FirefoxHeadless]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+    - name: Checking out Repository
+      uses: actions/checkout@v1
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Install Chrome (macOS)
+      if: matrix.os == 'macOS-latest' && matrix.browser == 'ChromeHeadless'
+      run: brew cask install google-chrome
+    - name: Install Firefox (macOS)
+      if: matrix.os == 'macOS-latest' && matrix.browser == 'FirefoxHeadless'
+      run: brew cask install firefox
+    - name: Install Dependencies
+      run: yarn bootstrap
+    - name: Run Tests
+      run: yarn test
+      env:
+        BROWSER: ${{ matrix.browser }}
+    - name: Upload Test Coverage to codecov
+      run: yarn run codecov
```

Comment:
Should we use the [CodeCov Action](https://github.com/codecov/codecov-action) in this job, rather than have the CI build run it locally?

Reply:
Regarding the token issue: https://community.codecov.io/t/whitelist-github-action-servers-to-upload-without-a-token/491/2

https://github.community/t5/GitHub-API-Development-and/Whitelist-Github-Workflow/td-p/15916

Reply:
https://github.com/react-bootstrap/react-overlays/pull/372#issuecomment-524907547